### ISSUE TITLE
Issue in combine two channel step in BAM

### ISSUE THREAD

**Issue by nguyen599** [Jongchan/attention-module#52]
Hi author, thank for good code. I confused in combine channel and spatial. In [your paper](https://arxiv.org/abs/1807.06514) before go to sigmoid func, you add two channel, but in code is multiply. Is it mistake?
![image](https://github.com/Jongchan/attention-module/assets/144032789/a2022db4-0183-4f5f-92d1-48a7110b61e2)

```
class BAM(nn.Module):
    def __init__(self, gate_channel):
        super(BAM, self).__init__()
        self.channel_att = ChannelGate(gate_channel)
        self.spatial_att = SpatialGate(gate_channel)
    def forward(self,in_tensor):
        att = 1 + F.sigmoid( self.channel_att(in_tensor) * self.spatial_att(in_tensor) ) # multiply in here
        return att * in_tensor
```
